---
graphs:
  original:
    plugins: {dsg_mesh: MeshPlugin}
    graph:
      type: GraphFromFile
      filepath: $(arg scene_graph_path)
      colormap_filepath: $(arg semantic_colormap)
  clustering:
    plugins: {segments: LLMLayerPlugin, objects: LLMLayerPlugin, interlayer_edges: InterlayerPlugin}
    graph:
      type: GraphFromZmq
      url: $(arg zmq_url)
gt_publisher:
  filepath: $(arg gt_path)
  scale: 0.05
  tasks:
    colormap_filepath: $(arg semantic_colormap)
    task_service_name: /gt_task_server/get_embedding
clustering:
  interlayer_edges:
    skiplist: [4, 5]
  segments:
    layer: 2
    layer_to_use: 2
    visualization_layer: 2
    min_saturation:  0.5
  objects:
    layer: 3
    layer_to_use: 3
    visualization_layer: 3
    min_saturation: 0.5
  config:
    layer2:
      visualize: true
      z_offset_scale: 1.5
      marker_scale: 0.05
      use_bounding_box: true
      use_sphere_marker: true
      collapse_bounding_box: false
      bbox_wireframe_scale: 0.02
      bbox_wireframe_edge_scale: 0.02
    layer3:
      visualize: true
      z_offset_scale: 0.5
      marker_scale: 0.3
      use_label: true
      label_scale: 0.25
      use_bounding_box: true
      collapse_bounding_box: true
      bbox_wireframe_scale: 0.02
      bbox_wireframe_edge_scale: 0.02
      interlayer_edge_scale: 0.01
      interlayer_edge_alpha: 0.6
